go/vt/vtadmin/api.go

```diff
@@ -208,7 +209,7 @@ func NewAPI(clusters []*cluster.Cluster, opts Options) *API {
 
 	if len(opts.HTTPOpts.CORSOrigins) > 0 {
 		serv.Router().Use(handlers.CORS(
-			handlers.AllowCredentials(), handlers.AllowedOrigins(opts.HTTPOpts.CORSOrigins)))
+			handlers.AllowCredentials(), handlers.AllowedOrigins(opts.HTTPOpts.CORSOrigins), handlers.AllowedMethods([]string{"GET", "HEAD", "POST", "PUT", "DELETE", "OPTIONS"})))
```

Comment:
I opted to not make these an HTTPOpts because they are necessary for parts of VTAdmin to function (like  RefreshState)